### DISCUSSION TITLE
regeneration de token enedis

### DISCUSSION THREAD

**Discussion by gypais** [bokub/conso-api#50] [Aide]
bonjour, j'autorise depuis plusieurs années enedis à suivre ma consommation, aussi la page 
https://mon-compte-particulier.enedis.fr/dataconnect... est déjà rempli et je ne peux pas refaire la procédure et donc générer le token.
Ou est stocké le token ou le refaire ?
Merci.

**Comment by bokub** [comment]
Salut @gypais 
Je comprend pas ce que tu veux dire par "déjà rempli". Pourquoi ne peux-tu pas refaire la procédure ?

**Comment by gypais** [comment]
Je viens de déactiver sur le site l'enregistrement pour refaire la procédure, mais maintenant le bouton pour le réactiver est grisé, je ne peux pas cliquer dessus :(
Peut-être faut-il attendre un peu ?

**Comment by gypais** [comment]
oui probablement, car il y a écrit "demande en cours", je pense que ca veut dire que ma demande de résiliation est en cours.

**Comment by bokub** [comment]
Désolé mais je comprend pas du tout de quoi tu parles, il va me falloir une capture d'écran sinon je vais pas pouvoir t'aider...

Normalement tu peux générer des tokens en quantité illimitée, que tu en aies déjà un ou non ne change rien au process

**Comment by gypais** [comment]
<img width="764" alt="Capture d’écran 2024-04-18 à 09 39 07" src="https://github.com/bokub/conso-api/assets/60287372/48217544-99c4-4d90-9352-a14bd34e5701">
voila mon écran, mais le bouton est grisé :(

**Reply by bokub** [reply]
Ah oui en effet, je pense que tu dois juste attendre que le "Demande en cours" disparaisse

**Reply by carlux3** [reply]
Bonjour, as tu pu récupérer ton token?

Je suis un peu dans la même situation : mes donnés étaient deja en partage et je n'arrive pas trouver l'endroit où je peux retrouver le token sur le site enedis, donc j'ai pensé suspendre le partage puis le réactiver pour voir si ça aidé !

**Reply by bokub** [reply]
> je n'arrive pas trouver l'endroit où je peux retrouver le token sur le site enedis

Tu ne peux pas retrouver un token que tu as perdu, tu dois forcément en re-générer un.

Malheureusement, le don de consentement est cassé en ce moment, donc tu ne peux rien faire pour l'instant (cf #49 )

**Reply by bokub** [reply]
C'est bon le consentement fonctionne à nouveau, vous devriez pouvoir re-générer vos tokens

Bonne journée

**Reply by carlux3** [reply]
Super ça a fonctionné, merci !
